interfaces/builtin/content.go

```diff
@@ -124,7 +123,12 @@ func (iface *contentInterface) path(slot *interfaces.Slot, name string) []string
 		panic("internal error, path can only be used with read/write")
 	}
 
-	paths, ok := slot.Attrs[name].([]interface{})
+	source, ok := slot.Attrs["source"].(map[string]interface{})
```

Comment:
I think it would be worth checking if the new syntax with "source" is not mixed with the old one ("read" or "write" at the same level as "source", in which case "source" would win and "old" ones would be silently ignored I think).

Reply:
Done in 2777b374e13081301d668b1332f6761a9f209046